### DISCUSSION TITLE
May I change openai api end point for AI assist panel?

### DISCUSSION THREAD

**Discussion by superwangnan** [zed-industries/zed#11305] [Feature Requests]
Hi team,
we setup a reverse proxy for internal use, so may I change the end point url in AI assist?
Thanks ~

**Comment by drcongo** [comment]
I'd like to know this one too. I have llama.cpp running a Codestral model on a Mac Studio in the corner of my office and I'd much rather be able to ask that than have anything to do with OpenAI.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
